pkg/kubelet/cm/cgroupManager_mock.go

```diff
@@ -0,0 +1,45 @@
+// +build linux
```

Comment:
I'm not sure why you'd need to use `MockCgroupManager` in `pkg/cm/*_test.go`  I think the unit tests in `pkg/cm` should be testing the actual implementation of the cgroup manager, while the mock will be used by unit tests in other packages (e.g., `kubelet_test.go`)